react/bingo/src/bingo/BingoApp.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BingoGame from './Game'
+import './BingoApp.css';
+
+class BingoApp extends React.Component {
+
+  constructor (props){
+    super(props);
+    this.state = {
+      challengeWord : '',
+      isGameOn : false
+    }
+  }
+
+  handleSubmit(){
+    this.setState ({
+      challengeWord : this.state.challengeWord,
+      isGameOn : true
+    });
+    playBingo(this.state.challengeWord);
+  }
+
+  refreshPage () {
```

Comment:
Resetting the page is achieved by setting the state (which triggers a re-render) variables to it's defaults.

Reply:
Need to work on this. Did not work initially.